lib/Settings/UserFlags.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace MailPoet\Settings;
+
+use MailPoet\Models\UserFlag;
+use MailPoet\WP\Functions as WPFunctions;
+
+class UserFlags {
+
+  /**
+   * @var array|null
+   */
+  private static $data = null;
+
+  public function getDefaults() {
+    return [
+      'last_announcement_seen' => false,
+      'editor_tutorial_seen' => false,
+    ];
+  }
+
+  public function load() {
+    $current_user_id = WPFunctions::get()->getCurrentUserId();
+    $flags = UserFlag::where('user_id', $current_user_id)->findMany();
+    self::$data = [];
+    foreach ($flags as $flag) {
+      self::$data[$flag->name] = $flag->value;
+    }
+  }
+
+  public function isLoaded() {
```

Comment:
Why is everything public? First of all AFAIK, our coding standards say we shouldn't write the `public` keyword. But most importantly we should really keep our method private as much as we can. Or other plugins will call them.